tests/save-images/new-behaviour-remote/Earthfile

```diff
@@ -1,5 +1,5 @@
 VERSION  --referenced-save-only 0.5
```

Comment:
```suggestion
VERSION  0.7
```

Reply:
I think we can leave this as-is since this test is specifically for this test; I think it makes it a bit more clear as to which feature we're testing new vs old.

Once `0.5` is no longer supported, we could rename the "new-behaviour` to a simple _only_ behaviour, and update it to 0.7 (or maybe even a larger version depending on where we are)